File: day_1/99_bottles.js
function bottles_function(bottle_count) {
  if (bottle_count <= 1) {
    console.log("Time to get more bottles!")
  }
  let song = ""
  let initial_count = bottle_count
  if (initial_count == 1) {
    initial_count = "1 bottle"
  } else {
    initial_count = initial_count.toString() + " bottles";
  }
  while (bottle_count >= 2) {
    let bottles
    bottles = bottle_count.toString() + " bottles";
    song += `${bottles} of beer on the wall, ${bottles} of beer.
`
    if (bottle_count ===2){
      bottles = "1 bottle"
    } else {
      bottles = (bottle_count -1).toString() + " bottles";
    }
    song +=`Take one down and pass it around, ${bottles} of beer on the wall.
`
    bottle_count -= 1
  }
  song += `1 bottle of beer on the wall, 1 bottle of beer.
Take one down and pass it around, no more bottles of beer on the wall.
No more bottles of beer on the wall, no more bottles of beer.
Go to the store and buy some more, ${initial_count} of beer on the wall.`
  console.log(song)
}

bottles_function(1)

File: day_1/99_bottles.py
def bottle_song(bottle_count): 
  if bottle_count < 1:
    return "Time to get more bottles!"
  song = "" 
  initial_count = bottle_count
  while bottle_count > 1:
    song += f"{bottle_count} bottles of beer on the wall, {bottle_count} bottles of beer.\n"
    bottle_count -=1
    if bottle_count > 1:
      song +=f"Take one down and pass it around, {bottle_count} bottles of beer on the wall.\n"
    else:
      song +=f"Take one down and pass it around, {bottle_count} bottle of beer on the wall.\n"
  if initial_count > 1:
    initial_count = str(initial_count) + " bottles"
  else:
    initial_count = "1 bottle"
  song += f"""1 bottle of beer on the wall, 1 bottle of beer.
Take one down and pass it around, no more bottles of beer on the wall.
No more bottles of beer on the wall, no more bottles of beer.
Go to the store and buy some more, {initial_count} of beer on the wall."""
  return song

print(bottle_song(int(input("Please provide the amount of bottles you have.\n"))))